src/table.scss

```diff
@@ -486,4 +486,15 @@ $block: #{$fd-namespace}-table;
       text-align: right;
     }
   }
+
+  .fd-object-identifier {
```

Comment:
We shouldn't be referencing other components like this in files - table should be adding a "table" class to object identifier.

Reply:
fixed